### PR TITLE
Capacity update (FR, ES, BE, FI, GB, KR)

### PR DESCRIPTION
**Pull request by gadakast**
Capacity update (FR, ES, BE, FI, GB, KR)

## Issue
Outdated data
## Description
Capacity updates for France, Spain and nuclear capacity updates for Belgium, Finland, GB and S. Korea. (Unfortunately, I wasn’t able to edit multiple files in one pull request, so the sources are on another one.)

### CODE REVIEW

File: config/zones.json [hunk 1]
```diff
@@ -449,12 +449,12 @@
       "geothermal": 0,
       "hydro": 177,
       "hydro storage": 1308,
-      "nuclear": 5943,
+      "nuclear": 4936,
```
Comment (VIKTORVAV99) [inline config/zones.json:452]: The source you provided in the other PR (IAEA) is showing an operational capacity of 5942 MW?
Please provide some clarity around the big difference here.

File: config/zones.json [hunk 2]
```diff
@@ -1601,19 +1601,19 @@
...
       "coal": 1818,
-      "gas": 12218,
+      "gas": 12752,
       "geothermal": 16,
       "hydro": 18743.6955,
       "hydro storage": 5029.87,
       "nuclear": 61370,
-      "oil": 2746,
-      "solar": 12447,
+      "oil": 2860,
```
Comment (VIKTORVAV99) [inline config/zones.json:1611]: The source seems to have a value of 2890 MW for oil.

**Review by VIKTORVAV99 — CHANGES_REQUESTED** [review]
I found at least 2 capacity updates that don't match what is in the source so please dubble check everything once more.

Beyond that it's great work!

File: config/zones.json (hunk 1 above)
Comment (gadakast) [reply config/zones.json:452]: Doel 3 reactor was permenantly shut down 09.23.2022. Source: https://afcn.fgov.be/fr/actualites/doel-3-arret-en-toute-surete According to the IAEA, its net capacity was 1006 MW, so the new nuclear capacity is now 4936 MW, not 5942 MW. Right now the IAEA’s 5942 MW nuclear capacity for Belgium is outdated too.

File: config/zones.json (hunk 2 above)
Comment (gadakast) [reply config/zones.json:1611]: Yes indeed, 2890 MW is the correct value.

File: config/zones.json (hunk 1 above)
Comment (VIKTORVAV99) [reply config/zones.json:452]: Then this should be included in the DATA_SOURCES.md file until IAEA is updated.

Comment (gadakast) [reply config/zones.json:452]: Correct, but at the time I made the change, Belgium’s Federal Agency for Nuclear Control hadn’t issued this official statement. So I will include this source too.

Comment (gadakast) [reply config/zones.json:452]: New source now included in the other PR.

File: config/zones.json (hunk 2 above)
Comment (gadakast) [reply config/zones.json]: Change made here too.

**Review by VIKTORVAV99 — APPROVED** [review]

**Comment by jarek** [comment]
Hello, I merged the changes to DATA_SOURCES.md from #4597 into this PR, so that they can be merged together and the change history is easier to track. I also resolved a merge conflict that resulted from PR #4601 changing the same file.

**Comment by VIKTORVAV99** [comment]
@gadakast This is merged now, thanks for the help!

**Pull request merged**